lib/HTTP/Response.pm

```diff
@@ -512,9 +531,17 @@ order):
 
 =item 1.
 
-A "Content-Disposition:" header in the response.  Proper decoding of
```

Comment:
@vanHoesel, updated documentation is here.

Reply:
I would write these lines as:

    my ($charset, $lang, $encfile) = $cd_param{'filename*'} =~ m/^([^']+)'(.*)'(.+)$/